synapse/cortex.py

```diff
@@ -6151,8 +6158,28 @@ async def disableCronJob(self, iden):
             iden (bytes):  The iden of the cron job to be changed
         '''
         await self.agenda.disable(iden)
+        await self._killCronTask(iden)
```

Comment:
should we also kill the task if the cronjob is deleted?

Reply:
Good call, thanks!